### DISCUSSION TITLE
圖片轉圖標的兩個問題

### DISCUSSION THREAD

**Discussion by alctny** [he3-app/he3-feedback#65] [Ideas]
功能：圖片轉圖標

問題：
1. 不勾選平台會生成空文件
2. 不輸入文件名默認生成的文件名會以`--`開頭，以 -- 開頭對命令行很不友好，且意義不明

希望解決方案：
1. 不勾選目標平台默認生成所有平台或進行提示
2. 默認文件名不要以 `--` 開頭

**Comment by hongchaodeng** [comment]
你好，非常感謝你的建議！我們會盡快修復。
